### PR TITLE
REST v1: collection types are not reported in table v1/kesyspaces/{ks}/tables/{t}

### PR DESCRIPTION
**Pull request by dougwettlaufer**

### CODE REVIEW

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
         

Clones added
============
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6302599&bid=20496339)

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
         

Clones added
============
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6302599&bid=20496339)

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
         

Clones added
============
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6302599&bid=20496339)

File: DEV_GUIDE.md [hunk 1]
```diff
@@ -133,3 +139,19 @@ curl -L -X GET 'localhost:8082/v1/keyspaces' \
 --header 'content-type: application/json' \
 --header 'X-Cassandra-Token: <AUTH_TOKEN>'
 ```
+
+## Running Tests
+
+Stargate includes a full suite of integration tests that can be ran in various ways. Some examples are
```
Comment (polandll) [inline DEV_GUIDE.md:145]:
```suggestion
Stargate includes a full suite of integration tests that can be ran in various ways. Some examples are:
```

Comment (dougwettlaufer) [reply DEV_GUIDE.md:145]: Thanks! Might have you look through the generated swagger UI later to help clean up.

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
         

Clones added
============
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6302599&bid=20496339)

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
         

Clones added
============
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6302599&bid=20496339)

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
         

Clones added
============
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6302599&bid=20496339)

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
         

Clones added
============
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6302599&bid=20496339)

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
         

Clones added
============
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6302599&bid=20496339)

**Comment by dougwettlaufer** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
         

Clones added
============
- testing/src/test/java/io/stargate/it/http/RestApiv2Test.java  1
- testing/src/test/java/io/stargate/it/http/RestApiTest.java  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/stargate/stargate/pullRequest?prid=6302599&bid=20496339)

**Review by EricBorczuk — APPROVED** [review]

**Pull request merged**